### PR TITLE
[13.0][MIG]hr_contract_document

### PR DESCRIPTION
**Pull request by alex-guardado**
Standard migration.

### CODE REVIEW

File: hr_contract_document/__manifest__.py [hunk 1]
```diff
@@ -3,14 +3,14 @@
 
 {
     "name": "HR Contract Document",
-    "version": "12.0.1.0.0",
+    "version": "13.0.1.0.0",
     "category": "Human Resources",
     "website": "https://github.com/OCA/hr",
     "author": "Brainbean Apps, " "Odoo Community Association (OCA)",
```
Comment (alexey-pelykh) [inline hr_contract_document/__manifest__.py]: `" "` could be removed

File: hr_contract_document/__manifest__.py [hunk 2]
```diff
@@ -3,14 +3,14 @@
...
     "license": "AGPL-3",
     "installable": True,
     "application": False,
     "summary": "Documents attached to the contact",
-    "depends": ["hr_contract", "document"],
+    "depends": ["hr_contract"],
```
Comment (alexey-pelykh) [inline hr_contract_document/__manifest__.py:14]: I'm a bit out of 13.0 context, please elaborate a bit more

**Review by tonygalmiche — APPROVED** [review]
LGTM : Functional review only with runbot

image

**Comment by jarroyomorales** [comment]
@alexey-pelykh @tonygalmiche Why not using the chatter attachments instead of this module?

**Comment by tonygalmiche** [comment]
> @alexey-pelykh @tonygalmiche Why not using the chatter attachments instead of this module?

I have no opinion on the usefulness of this module. I just check that it works :)

**Comment by alex-guardado** [comment]
> @alexey-pelykh @tonygalmiche Why not using the chatter attachments instead of this module?

This module is not a 'variation' of chatter attachments. It simply 'extends' the chatter attachments adding a smart button up to the right to make documents more accesible.

File: hr_contract_document/__manifest__.py (hunk 1 above)
Comment (alex-guardado) [reply hr_contract_document/__manifest__.py]: I've just changed. Thank you! :)

**Review by misern2 — APPROVED** [review]
LGTM. Tested on runbot

:+1:

**Review by jarroyomorales — APPROVED** [review]

**Review by Saran440 — APPROVED** [review]
Functional Test :+1:

**Pull request merged**